src/lib/signature_lib/checked.ml

```diff
@@ -154,12 +163,15 @@ module Schnorr
         val zero : t
 
         val add : t -> t -> t
+        val subtract : t -> t -> t
 
         val double : t -> t
 
         val scale : t -> Scalar.t -> t
 
         val to_affine_coordinates : t -> Field.t * Field.t
+        val is_on_curve : t -> bool
+        val is_inf : t -> bool
```

Comment:
I think we can just do `equal zero` no ?

Reply:
yeah, these are not used